src/Umbraco.Web.UI.Client/src/packages/documents/documents/collection/action/create-document-collection-action.element.ts

```diff
@@ -56,20 +56,20 @@ export class UmbCreateDocumentCollectionActionElement extends UmbLitElement {
 
 	override async firstUpdated() {
 		if (this._documentTypeUnique) {
-			this.#retrieveAllowedDocumentTypesOf(this._documentTypeUnique);
+			this.#retrieveAllowedDocumentTypesOf(this._documentTypeUnique, this._documentUnique || null);
 		}
```

Comment:
And here, I would suggest firing a proper Error, describing why it did not work.

like: "Could not retrieve Document Type Unique or Parent Document Unique"